Gemfile.lock

```diff
@@ -0,0 +1,121 @@
+PATH
```

Comment:
I believe it's best practice to check this in for applications but not rubygems that need to work in a variety of different environments/ruby versions.